api/SubtleCrypto.json

```diff
@@ -670,6 +670,55 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "ed25519": {
+          "__compat": {
+            "description": "<code>Ed25519</code> algorithm",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/SubtleCrypto/sign",
+            "spec_url": "https://w3c.github.io/webcrypto/#SubtleCrypto-method-sign",
```

Comment:
Wouldn't it be better to link to https://wicg.github.io/webcrypto-secure-curves/#ed25519?